src/stores/UiStore.js

```diff
@@ -0,0 +1,38 @@
+import { observable, action, reaction, computed } from 'mobx';
+import moment from 'moment';
+import locales from '../languageProvider';
+
+
+export default class UIStore {
```

Comment:
nit: what do you think about calling it `LangStore`? or too specific?

Reply:
if wont change, can we rename to `UiStore`? read somewhere that its normal to use camelcase even for abbreviated words.

Reply:
Gah, I totally thought I made it `UiStore` lol. I wrote it `UIStore` at first but I thought I switched it 🤦‍♂️. So the reason I chose `UiStore` was from [this part of the docs](https://mobx.js.org/best/store.html). We can do `LangStore` but yeah I think it might be too specific

Reply:
Oh, I just realized I did `UiStore` everywhere except the definition name. Good catch